tests/test_dq.py

```diff
@@ -1,8 +1,12 @@
+import pytest
+from astropy.nddata.bitmask import interpret_bit_flags as ap_interpret_bit_flags
 from stdatamodels import dqflags
 from stdatamodels.jwst.datamodels.dqflags import pixel
 
 
 def test_dqflags():
     assert dqflags.dqflags_to_mnemonics(1, pixel) == {"DO_NOT_USE"}
     assert dqflags.dqflags_to_mnemonics(7, pixel) == {"JUMP_DET", "DO_NOT_USE", "SATURATED"}
-    assert dqflags.interpret_bit_flags("DO_NOT_USE + WARM", mnemonic_map=pixel) == 4097
+    with pytest.warns(DeprecationWarning):
+        assert dqflags.interpret_bit_flags("DO_NOT_USE + WARM", mnemonic_map=pixel) == 4097
+    assert ap_interpret_bit_flags("DO_NOT_USE+WARM", flag_name_map=pixel) == 4097
```

Comment:
Maybe not something as serious as conversion guide but this whitespace business should be documented somewhere for users looking to switch over? Is string usage with operator common?

Reply:
let me know if https://github.com/spacetelescope/stdatamodels/pull/486/commits/3655c3bf89365c67edd66defb152b3f0a0d8ddaa looks ok